packages/app/components/TokenList/Item.tsx

```diff
@@ -19,21 +18,21 @@ interface Props {
 
 export const TokenItem: FC<Props> = ({ style, item }) => {
 	const { metadata = {}, account } = item;
-	const { name, symbol, imageUri } = metadata;
+	const { symbol, imageUri } = metadata;
 	const amount = parseTokenAccount(account);
 	const unitQuote = account.quotes?.usd;
 	const totalQuote = unitQuote && unitQuote * amount;
 	const iconSource = {
-		uri: imageUri || '/img/question.png',
+		uri: imageUri ?? '/img/send-token/unknown-token.jpeg',
```

Comment:
`??` only work if `imageUri` is null/undefined, if it is falsy (i.e empty string), uri will take empty string, please carefully consider this situation